src/main/java/Duke.java

```diff
@@ -12,52 +10,48 @@ public static void main(String[] args) {
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
 
-        System.out.println("____________________________________________________________\n" +
-                "Hello! I'm Duke\n" +
-                "What can I do for you?\n" +
-                "____________________________________________________________\n");
+        System.out.println("____________________________________________________________\n"
+                + "Hello! I'm Duke\n"
+                + "What can I do for you?\n"
+                + "____________________________________________________________\n");
         Scanner input = new Scanner(System.in);
-        while (true){
+        while (true) {
             String str = input.nextLine();
-            if (str.equals("bye")){
-                System.out.println("____________________________________________________________\n" +
-                        "Bye. Hope to see you again soon!\n" +
-                        "____________________________________________________________\n");
+            if (str.equals("bye")) {
+                System.out.println("____________________________________________________________\n"
+                        + "Bye. Hope to see you again soon!\n"
+                        + "____________________________________________________________\n");
                 break;
             }
-            else if (str.equals("list")){
-                System.out.println("____________________________________________________________\n" +
-                        "Here are the tasks in your list: ");
-                list.printItems();
+            else if (str.equals("list")) {
+                System.out.println("____________________________________________________________\n"
+                        + "Here are the tasks in your list: ");
+                List.printItems();
                 System.out.println("____________________________________________________________");
             }
```

Comment:
The bracket for the if-else statement should be on the same line as else, so the line should be : 
} else {